jobrunner/config.py

```diff
@@ -255,7 +255,10 @@ def parse_job_resource_weights(config_file):
 # BindMountVolumeAPI config
 #
 # used to store directories to be mounted into jobs with the BindMountVolumeAPI
-HIGH_PRIVACY_VOLUME_DIR = HIGH_PRIVACY_STORAGE_BASE / "volumes"
+HIGH_PRIVACY_VOLUME_DIR = os.environ.get(
+    "HIGH_PRIVACY_VOLUME_DIR",
+    HIGH_PRIVACY_STORAGE_BASE / "volumes",
+)
```

Comment:
It would be worth wrapping this in `Path()` otherwise you'll get a `Path` as the default and then a string if the env var is set which is liable to cause problems.